doc/getting_started.rst

```diff
@@ -0,0 +1,185 @@
+``BaseEstimator`` class). These objects don't have a :term:`predict` method
+but rather a :term:`transform` method that outputs a newly transformed
+sample matrix ``X``::
+
+  >>> from sklearn.preprocessing import StandardScaler
+  >>> X = [[0, 15],
+  ...      [1, -10]]
+  >>> StandardScaler().fit(X).transform(X)
+  array([[-1.,  1.],
+         [ 1., -1.]])
+
+Transformers and estimators (predictors) can be combined together into a single
+unifying object: a :class:`~sklearn.pipeline.Pipeline`.
+
+In the following example, we :ref:`load the Iris dataset <datasets>`, split it
+into train and test sets, and compute its accuracy score on the test data. The
+pipeline offers the same API as a regular estimator: it can be fitted and
+used for predictions with ``fit`` and ``predict``::
+
+  >>> from sklearn.preprocessing import StandardScaler
+  >>> from sklearn.linear_model import LogisticRegression
+  >>> from sklearn.pipeline import make_pipeline
+  >>> from sklearn.datasets import load_iris
+  >>> from sklearn.model_selection import train_test_split
+  >>> from sklearn.metrics import accuracy_score
+  ...
+  >>> preprocessor = StandardScaler()
+  >>> predictor = LogisticRegression(random_state=0)
+  >>> pipe = make_pipeline(preprocessor, predictor)
+  ...
+  >>> X, y = load_iris(return_X_y=True)
+  >>> X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=0)
+  >>> pipe.fit(X_train, y_train)  # fit the preprocessor first, then the predictor
+  Pipeline(steps=[('standardscaler', StandardScaler()),
+                  ('logisticregression', LogisticRegression(random_state=0))])
+
+  >>> accuracy_score(pipe.predict(X_test), y_test)
+  0.97...
+
+Model evaluation
+----------------
+
+Once a model has been fitted, it is natural to evaluate its performance on
+unseen data. We have just seen the
+:func:`~sklearn.model_selection.train_test_split` helper, but scikit-learn
+provides many other tools for model evaluation, in particular for
+:ref:`cross-validation <cross_validation>`.
+
+We here briefly show how to perform a 3-folds cross-validation using the
+:func:`~sklearn.model_selection.cross_validate` helper. Note that it is also
+possible to manually iterate over the folds, and to use different data
+splitting strategies. Please refer to our :ref:`User Guide
+<cross_validation>` for more details.
+
+  >>> from sklearn.datasets import make_regression
+  >>> from sklearn.linear_model import LinearRegression
+  >>> from sklearn.model_selection import KFold
+  >>> from sklearn.model_selection import cross_validate
+  ...
+  >>> X, y = make_regression(n_samples=1000, random_state=0)
+  >>> lr = LinearRegression()
+  >>> cv = KFold(n_splits=3)
+  ...
+  >>> result = cross_validate(lr, X, y, cv=cv)
+  >>> result['test_score']
+  array([1., 1., 1.])
+
+Automatic parameter searches
+----------------------------
+
+All estimators have parameters that can be tuned for the predictions to be as
+good as possible. For example a
```

Comment:
This is more about better generalization of the model on the unseen data, not better predictions on the data we have, which is not clear from the sentence now.